index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>FizzBuzz問題</title>
+  <link rel="stylesheet" href="./issue/assets/css/style.css">
+ 
+</head>
+<body>
+  <h1>FizzBuzz問題</h1>
+
+  <div>
+    <div class="">
+      <label for="fizznum">FizzNum:</label>
+      <input type="text" id="fizzNum" name="FizzNum">
+    </div>
+
+    <div>
+      <label for="buzzNum">BuzzNum:</label>
+      <input type="text" id="buzzNum" name="buzzNum">
+    </div>
+
+    <button id="runButton">実行</button>
+
+    <div id="resultArea">[出力]</div>
```

Comment:
idやclassなどの命名に関して
HTML側では、複数の単語の場合は`-`で単語を繋ぐケバブケースと呼ばれる規則の倣うのが一般的です。
例えばこちらですと以下のようになります！
```suggestion
    <div id="result-area">[出力]</div>
```
他の箇所も同様に修正してみましょう！